test/teslamate/vehicles/vehicle/charging_test.exs

```diff
@@ -54,6 +55,7 @@ defmodule TeslaMate.Vehicles.Vehicle.ChargingTest do
                       date: _,
                       charge_energy_added: 0.2,
                       rated_battery_range_km: 3.22,
+                      charge_rate: 2,
```

Comment:
these values should set to realisitc values

Reply:
They're , 2km/h would be equivalent to 300w, attached you can see a 32Amp pull from a charger (7.4kW) but as preconditioning is on, only mere 240W are going to the battery (1 mi/h in this case)
![IMG_2895](https://github.com/user-attachments/assets/282b0a67-44ad-4cbe-9f06-a9ef0e9f866f)

Reply:
as before, `charge_rate` unit is not obvious